doc/nrf/gs_installing.rst

```diff
@@ -695,3 +564,152 @@ Define the required environment variables as follows, depending on your operatin
 You must also make sure that nrfjprog (part of the `nRF Command Line Tools`_) is installed and its path is added to the environment variables.
 The west command programs the kit by using nrfjprog by default.
 For more information on nrfjprog, see `Programming SoCs with nrfjprog`_.
+
+.. _gs_updating:
+
+Updating tools and repositories
+*******************************
+
+Remember to regularly check for updates to tools.
+Like any other Python package, the west tool is updated regularly.
+
+You might also want to switch to a newer release or check out the latest state of development.
+However, if you work with a specific release of the |NCS|, you do not need to update your repositories, because the release will not change.
+
+.. _west_update:
+
+Updating west
+=============
+
+To update west, run the following command:
+
+.. tabs::
+
+   .. group-tab:: Windows
+
+      .. parsed-literal::
+         :class: highlight
+
+         pip3 install -U west
```

Comment:
Does this always update west to its latest version? I updated mine with pip install west == 0.10.0, for example.

Reply:
To its latest version available in PyPI.